modules/core/src/main/scala/org/scalasteward/core/nurture/NurtureAlg.scala

```diff
@@ -84,8 +84,8 @@ final class NurtureAlg[F[_]](
   def processUpdate(data: UpdateData): F[Unit] =
     for {
       _ <- logger.info(s"Process update ${data.update.show}")
-      head = vcs.headFor(data.fork.show, data.update)
-      pullRequests <- gitHubApiAlg.listPullRequests(data.repo, head, data.baseBranch)
+      head = vcsSpecifics.sourceFor(data.repo, data.update)
```

Comment:
This does not do the right thing for GitHub repos since `head` is not created from `data.fork` (see #552 why this is needed). I'll try to change this later.